test/library/packages/Sort/RadixSort/distSortEg.chpl

```diff
@@ -0,0 +1,32 @@
+use BlockDist;
+use Random;
+use Sort;
+use simpleMerge;
+
+config const n = 10;
+const Space = {1..n};
```

Comment:
As a general good practice, I'd put `Space` in `main`. If you define a main function explicitly, I think global variables should be avoided as much as possible.